packages/tools/eslint-config-custom/index.js

```diff
@@ -36,7 +36,15 @@ module.exports = {
     "@typescript-eslint/explicit-function-return-type": "error",
     "@typescript-eslint/no-explicit-any": "error",
     "import/default": "off",
-    eqeqeq: ["error", "always"]
+    eqeqeq: ["error", "always"],
+    "no-restricted-syntax": [
+      "error",
+      {
+        selector:
+          "CallExpression[callee.name='describe'] MemberExpression[object.type='ThisExpression'][property.name='timeout']",
+        message: "Manual timeouts in Mocha tests are not allowed."
+      }
```

Comment:
this is so cool i'm going to ban so much stuff

Reply:
https://eslint.org/play is great for testing these things interactively